commands/show_test.go

```diff
@@ -59,48 +71,69 @@ func TestBlockDaemon(t *testing.T) {
 func TestShowDeal(t *testing.T) {
 	tf.IntegrationTest(t)
 
-	minerDaemon := th.NewDaemon(t,
-		th.WithMiner(fixtures.TestMiners[0]),
-		th.KeyFile(fixtures.KeyFilePaths()[0]),
-		th.DefaultAddress(fixtures.TestAddresses[0]),
-		th.AutoSealInterval("1"),
-	).Start()
-	defer minerDaemon.ShutdownSuccess()
+	fastenvOpts := fast.EnvironmentOpts{
+		//InitOpts: []fast.ProcessInitOption{fast.POAutoSealIntervalSeconds(1)},
+		//DaemonOpts: []fast.ProcessDaemonOption{fast.POBlockTime(time.Second)},
+	}
 
-	clientDaemon := th.NewDaemon(t,
-		th.KeyFile(fixtures.KeyFilePaths()[1]),
-		th.DefaultAddress(fixtures.TestAddresses[1]),
-	).Start()
-	defer clientDaemon.ShutdownSuccess()
+	ctx, env := fastesting.NewTestEnvironment(context.Background(), t, fastenvOpts)
 
-	minerDaemon.RunSuccess("mining", "start")
-	minerDaemon.UpdatePeerID()
+	clientNode := env.GenesisMiner
+	require.NoError(t, clientNode.MiningStart(ctx))
```

Comment:
We need to figure out what we want to do about registering the mining protocol without running MiningStart. It'd be nice if we could avoid starting it.